service/index.js

```diff
@@ -0,0 +1,29 @@
+const Contact = require("./schemas/contacts");
+
+const getAll = async () => {
+  return Contact.find({}).lean();
+};
+
+const getById = (id) => {
+  return Contact.findById({ _id: id });
+};
+
+const createNew = ({ name, email, phone }) => {
+  return Contact.create({ name, email, phone });
+};
```

Comment:
jezeli nie przyjmujesz niczego innego, jak tylko `name`, `email` i `phone`, to nie ma sensu destrukturyzowac. A powinienes generalnie przekazywac tylko to, co jest Ci potrzebne. Wiec mozesz znacznie prosciej:
`const createNew = (c) => Contact.create(c);`

Reply:
Done